src/item.cpp

```diff
@@ -4483,6 +4486,23 @@ bool item::conductive() const
     } );
 }
 
+bool item::reinforceable() const
+{
+    if( is_null() ) {
+        return false;
+    }
+
+    if( has_flag( "NO_REPAIR" ) ) {
+        return true;
```

Comment:
This will allow reinforcing some non-cloth items such as dragonskin vest. Maybe you meant `false`?

Reply:
Oh, oops. Thanks for spotting that. Yeah, it's a false.